src/VisualStudio/Core/Def/Implementation/TaskList/CommentTaskTokenSerializer.cs

```diff
@@ -22,11 +23,11 @@ internal class CommentTaskTokenSerializer : IOptionSerializer
 
         [ImportingConstructor]
         public CommentTaskTokenSerializer(
-            SVsServiceProvider serviceProvider, IOptionService optionService)
+            VisualStudioWorkspaceImpl workspace)
         {
-            _optionService = optionService;
+            _optionService = workspace.Services.GetService<IOptionService>();
```

Comment:
I guess I imagined this would pull in the global option service now, so it's clear it's no targeting this one specific workspace?